rotkehlchen/chain/ethereum/modules/eth2/eth2.py

```diff
@@ -136,7 +136,7 @@ def fetch_eth1_validator_data(
             self,
             addresses: list[ChecksumEvmAddress],
     ) -> list[ValidatorID]:
-        """Query all eth1 addresses for their validators and get all corresponding deposits.
+        """Query all eth1 addresses for their validators.
```

Comment:
Also perhaps rename function to show you are writing to the DB. `fetch_and_update_eth1_validator_data`

```suggestion
        """Query all eth1 addresses for their validators and any newly detected validators are added to the DB
```